javascript/src/index.ts

```diff
@@ -875,3 +881,25 @@ export class Webhook {
     return new Date(timestamp * 1000);
   }
 }
+
+class Statistics {
+  private readonly api: StatisticsApi;
+
+  public constructor(config: Configuration) {
+    this.api = new StatisticsApi(config);
+  }
+
+  public aggregateEventTypes(): Promise<AggregateEventTypesOut> {
+    return this.api.aggregateEventTypes({});
+  }
+
+  public calculateAggregateAppStats(
```

Comment:
```suggestion
  public aggregateApplicationMessages(
```
Stats and calculate are redundant. Need to make sure the event type name is also correct for this, but this should be the name of the function all around.

Reply:
Shall I change the name in our API docs then as well?

Reply:
Yup, but that's a one-line change, no?